manim/utils/color.py

```diff
@@ -10,7 +10,7 @@
 from ..utils.space_ops import normalize
 
 
-def color_to_rgb(color):
+def color_to_rgb(color: Color):
```

Comment:
return type?

Reply:
I know it returns an array of either ints or floats, but I couldn't figure out which